contrib/time_series_expectations/docs/README.md

```diff
@@ -0,0 +1,176 @@
+
+* ...
+* https://www.machinelearningplus.com/time-series/arima-model-time-series-forecasting-python/
+* https://neptune.ai/blog/arima-vs-prophet-vs-lstm
+
+
+You can install any of those packages like this: `pip install time_series_expectations[prophet, statsmodels]`
+
+Technically, `pip install time_series_expectations` is also allowed. But if you don't specify a time series dependency at all, very few of the Expectations in this package will work.
+
+
+### Data Assistants and Usage
+
+This module contains four new DataAssistants, which serve as the primary entry points for the package.
+
+Warning: These docs assume that you have completed the Great Expectations onboarding tutorial, and are familiar with basic concepts including [Expectations](link), [DataAssistants](link), and [DataSources](link).
+
+* Freshness : "I always want to know how fresh my data is"
+
+```
+freshness_expectation_suite = context.assistants.freshness.profile_data_asset(
+    context.sources.my_db.assets.my_table
+)
+```
+
+* Volume: "I want to know if data is arriving at the expected rate"
+
+```
+volume_expectation_suite = context.assistants.volume.profile_data_asset(
+    context.sources.my_db.assets.my_table
+)
+```
+
+* BatchLevelTimeSeries: "I have a data asset with multiple batches arriving over time. I want to learn what patterns are normal in that data, and then monitor it for anomalies"
+
+```
+time_series_expectation_suite = context.assistants.time_series_by_batch.profile_data_asset(
+    context.sources.my_db.assets.my_table,
+    columns=["foo", "bar", "baz"],
+    metrics=["mean", "median", "sum", "percent_null"]
+)
+
+time_series_expectation_suite = context.assistants.time_series_by_batch.profile_data_asset(
+    context.sources.my_db.assets.my_table,
+    columns=["foo", "bar", "baz"],
+    metric="percent_matching_regex",
+    kwargs={
+        "regex": "\d+",
+    }
+)
+```
+
+
+* RowLevelTimeSeries: "I have numeric time series data in a table, file, etc. I want to learn what patterns are normal in that data, and then monitor it for anomalies"
+
+time_series_expectation_suite = context.assistants.time_series_by_row.profile_data_asset(
+    context.sources.my_db.assets.my_table,
+    date_column="created_at",
+    columns=["foo", "bar", "baz"],
+    metrics=["mean", "median", "sum", "percent_null"]
+)
+
+### New Expectations
+
+This module introduces several new Expectations, inheriting from some new abstract base classes.
+
+You can learn more about the Expectations in the Expectation gallery, [here](link).
+
+
+### Abstract Base Classes
+
+The most important ABCs are [BatchAggregateStatisticTimeSeriesExpectation](link), [ColumnAggregateTimeSeriesExpectation](link), and  [ColumnPairTimeSeriesExpectation](link). They allow time series models to be applied to data in a variety of shapes and formats. Please see the class docstrings for more detailed explanation.
```

Comment:
I love the code reuse and the class hierarchy here! This really makes the expectations simpler to build. I wonder though if AbstractBaseClass is the right name - it looks like the classes marked ABC are expectations that can be validated against, is that the case? If so maybe we can rename to something like `Parent Class` instead of ABC.

Reply:
I've moved this to the main README and clarified that these are actually ABCs.